client/starwhale/base/view.py

```diff
@@ -232,15 +232,19 @@ def list_data(
                 if _v.get("id"):
                     _version = f"[{_v['id']:2}] {_version}"
 
-                result.append(
-                    {
-                        "name": _name,
-                        "version": _version,
-                        "tags": _v.get("tags", []),
-                        "size": _v.get("size", 0),
-                        "created_at": _v.get("created_at"),
-                    }
-                )
+                _item = {
```

Comment:
```python
_item = {
                    "name": _name,
                    "version": _version,
                    "tags": _v.get("tags"),
                    "alias": _v.get("alias"),
                    "size": _v.get("size", 0),
                    "created_at": _v.get("created_at"),
                }
                if "alias" not in _v:
                    _item.pop("alias")
                if "tags" not in _v:
                    _item.pop("tags")
```
-->
```python
_item = {
                    "name": _name,
                    "version": _version,
                    "size": _v.get("size", 0),
                    "created_at": _v.get("created_at"),
                }
                if "alias" in _v:
                    _item["alias"] = _v["alias"]
                if "tags" in _v:
                    _item["tags"] = _v["tags"]
```
?

Reply:
327-339 lines do the same change?

Reply:
The purpose of this practice is to display the fields in the order given. Would you have a better way?

Reply:
python dict type is unordered.